src/token/ValidatorRewarder.sol

```diff
@@ -27,17 +31,13 @@ contract ValidatorRewarder is IValidatorRewarder, UUPSUpgradeable, OwnableUpgrad
     /// @notice The token that this rewarder mints
     Recall public token;
 
-    /// @notice The bottomup checkpoint period for the subnet.
-    /// @dev The checkpoint period is set when the subnet is created.
-    uint256 public checkpointPeriod;
-
     /// @notice The number of blocks required to generate 1 new token (with 18 decimals)
     uint256 public constant BLOCKS_PER_TOKEN = 3;
 
     // ========== EVENTS & ERRORS ==========
 
     event ActiveStateChange(bool active, address account);
-    event SubnetUpdated(SubnetID subnet, uint256 checkpointPeriod);
+    event SubnetUpdated(SubnetID subnet);
```

Comment:
Stop storing `checkpointPeriod`, we will fetch the number of blocks in a checkpoint on the fly